tests/unit/config/test_review_config.py

```
@@ -0,0 +1,175 @@
+  checklist:
+    items:
+      - not-a-mapping
+""",
+        encoding="utf-8",
+    )
+    monkeypatch.chdir(tmp_path)
+    clear_config_cache()
+
+    with pytest.raises(
+        ValueError,
+        match="review.checklist.items entries must be mappings",
```

Ruff (0.15.20)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)